pkgs/gcc/gcc.yaml

```diff
@@ -12,6 +12,14 @@ sources:
   url: http://ftpmirror.gnu.org/gcc/gcc-4.9.2/gcc-4.9.2.tar.bz2
 
 build_stages:
+  - name: rpath_patch
+    before: configure
+    files: [rpath.patch]
+    handler: bash
+    bash: |
+      patch -up1 < _hashdist/rpath.patch
+      sed -i "s|@@ARTIFACT@@|${ARTIFACT}|g" gcc/config/i386/gnu-user.h gcc/config/i386/gnu-user64.h
```

Comment:
@ahmadia I've searched the web and there doesn't seem to be any solution really to this, except to manually escape the variable to be substituted. I think it will also fail if the variable contains `[]`. I am inclined to just leave this be (it should work for most paths), unless we can figure out a better solution.